src/zcl/definition/cluster.ts

```diff
@@ -3842,6 +3842,68 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    heimanSpecificPM25Measurement: {
+        // from HS2AQ-3.0海曼智能空气质量检测仪API文档-V01
+        ID: 0x042a,
```

Comment:
Please add the `manufacturerCode` like in https://github.com/Koenkk/zigbee-herdsman/pull/222/files#diff-8befd3fb846239a5084337fd3cc8e083R3833

Reply:
Done.